lua/packer.lua

```diff
@@ -8,6 +8,7 @@ local stdpath = vim.fn.stdpath
 -- Config
 local packer = {}
 local config_defaults = {
+  snapshot_path = join_paths(stdpath 'cache', 'packer'),
```

Comment:
For consistency, this should probably default to `join_paths(stdpath 'cache', 'packer.nvim')`